### PR TITLE
Add pyupgrade to pre-commit checks

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: skyportal/models/instrument.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def bind_expression(self, bindvalue):
         return cast(bindvalue, self)
 
     def result_processor(self, dialect, coltype):
-        super_rp = super(ArrayOfEnum, self).result_processor(dialect, coltype)
+        super_rp = super().result_processor(dialect, coltype)
```
Comment (mcoughlin) [inline skyportal/models/instrument.py:29]: Is it surprising that this gives the same result as before?

**Review by mcoughlin — APPROVED** [review]
Crazy how many ways there are to do the same thing in python.

**Comment by stefanv** [comment]
> Crazy how many ways there are to do the same thing in python.

```
import this
```

> ...
> There should be one-- and preferably only one --obvious way to do it.
> Although that way may not be obvious at first unless you're Dutch.

**Review by guynir42 — APPROVED** [review]
LGTM

**Pull request merged**